chain/ethereum/src/adapter.rs

```diff
@@ -158,8 +158,10 @@ impl bc::TriggerFilter<Chain> for TriggerFilter {
 
     fn to_firehose_filter(self) -> Vec<prost_types::Any> {
         let EthereumBlockFilter {
+            polling_intervals: _polling_intervals,
             contract_addresses: _contract_addresses,
             trigger_every_block,
+            ..
```

Comment:
We should not use `..` here, as it makes it easy to add a field in the future without considering the consequences to this fn.